api/client/proto/authservice.proto

```diff
@@ -1497,6 +1500,8 @@ message ListResourcesResponse {
     // continue retrieving pages of resource. If NextKey is empty, there are no
     // more pages.
     string NextKey = 2 [ (gogoproto.jsontag) = "next_key,omitempty" ];
+    // TotalCount is the total number of resources available after filter, if any.
```

Comment:
"if any" but also if the total count was asked for and the request was legitimate?